pkg/webhook/default_server/pod/mutating/pod_create_handler.go

```diff
@@ -92,9 +95,19 @@ func (h *PodCreateHandler) sidecarsetMutatingPod(ctx context.Context, pod *corev
 		}
 		matchNothing = false
 
-		sidecarSetGeneration[sidecarSet.Name] = sidecarSet.Generation
+		hash, err := mutating.SidecarSetHash(&sidecarSet)
```

Comment:
Can you directly read the hash from sidecarset annotation instead of recomputing it from spec? 
The same for the SidecarsetHashWithoutImage.

Reply:
done